### PR TITLE
Adapt to foundry

### PR DESCRIPTION
**Pull request by MSNTCS**
This patch adapts codechain-rpc-js for foundry. Remove rpcs that are not
implemented in foundry. It fixes issue `#107` of foundry.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
   },
   "dependencies": {
     "@types/node-fetch": "^2.3.4",
+    "codechain-rpc": "^2.0.0",
```
Comment (majecty) [inline package.json]: Why we need to use codechain-rpc as dependency?

**Review by majecty — CHANGES_REQUESTED** [review]
The package name is "codechain-rpc". Please change it. 
The package name is described in the "name" field in package.json file.
Also, please search whole files to check whether there are "codechain-rpc" texts or not.

I recommend you to study the "package.json" format.
https://docs.npmjs.com/creating-a-package-json-file

**Comment by MSNTCS** [comment]
> The package name is "codechain-rpc". Please change it.
> The package name is described in the "name" field in package.json file.
> Also, please search whole files to check whether there are "codechain-rpc" texts or not.
> 
> I recommend you to study the "package.json" format.
> https://docs.npmjs.com/creating-a-package-json-file

Done ✅

File: package.json [hunk 2]
```diff
@@ -40,6 +43,12 @@
   },
   "dependencies": {
     "@types/node-fetch": "^2.3.4",
+    "foundry-rpc": "^2.0.0",
```
Comment (majecty) [inline package.json]: Why did you add this line?

File: package.json [hunk 3]
```diff
@@ -40,6 +43,12 @@
...
     "node-fetch": "^2.6.0"
-  }
+  },
+  "bugs": {
+    "url": "https://github.com/CodeChain-io/foundry-rpc-js/issues"
+  },
+  "homepage": "https://github.com/CodeChain-io/foundry-rpc-js#readme",
+  "keywords": []
```
Comment (majecty) [inline package.json:52]: Good!

**Review by majecty — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (MSNTCS) [reply package.json]: I did not add this, I think `npm init` do that.

Comment (majecty) [reply package.json]: Let's remove it.
All changed content must be made with your intent.

At least, you should know the effect of changed content.

**Review by majecty — APPROVED** [review]
